src/search/Drawer.jsx

```
@@ -60,6 +61,27 @@ class Drawer extends React.Component {
     } = this.props;
     const searchId = geofence.id || geofence.name;
 
+    RestAPI.requestTotalAreaInGeofence(area.id, searchId)
+      .then((res) => {
+        this.setState(prevState => ({
+          ...prevState,
+          data: {
+            ...prevState.data,
+            generalArea: Number(res.total_area),
+          },
+        }));
+      })
+      .catch(() => {
+        this.setState(prevState => ({
+          ...prevState,
+          data: {
+            ...prevState.data,
+            generalArea: 0,
```

This setState in the catch isn't necessary since the initial value for generalArea is 0

---

Should I remove all the content in the catch, as an empty catch?

---

yeah, i we have an error handler that would be different, but in this case you just don't need to do anything

---

Done.